java/arcs/core/storage/ActiveStore.kt

```diff
@@ -38,7 +38,7 @@ abstract class ActiveStore<Data : CrdtData, Op : CrdtOperation, ConsumerData>(
     abstract fun on(callback: ProxyCallback<Data, Op, ConsumerData>): Int
 
     /** Unregisters a callback associated with the given [callbackToken]. */
-    abstract fun off(callbackToken: Int)
+    abstract suspend fun off(callbackToken: Int)
```

Comment:
This asymmetry feels odd, but I didn't want to make an uneeded change to the on method, for now. I'm definitely open to debate about this API change.

Reply:
I think it's fine. Let's wait to change 'on' until we need to.